README.md

```diff
@@ -97,68 +97,121 @@ Most of the data currently being saved in Bonsai is packaged in a HARP message f
+|                           |       Longitude       |   228    |      1 Hz     |  Depends on having GPS signal                 |
+|                           |        Altitude       |   229    |      1 Hz     |  Depends on having GPS signal                 |
+|                           |          Data         |   230    |      1 Hz     |  Date from tinkerforge GPS device             |
+|                           |          Time         |   231    |      1 Hz     |  Time from tinkerforge GPS device             |
+|                           |         HasFix        |   232    |      1 Hz     |  Depends on having GPS signal                 |
+|        **TK-CO2V2**       |        CO2Conc        |   224    |       -       |  This sensor is not being used                |
+|                           |      Temperature      |   225    |       -       |  This sensor is not being used                |
+|                           |        Humidity       |   226    |       -       |  This sensor is not being used                |
+|    **TK-AmbientLight**    |      AmbientLight     |   223    |       -       |  This sensor is not being used                |
+|     **TK-AirQuality**     |       IAQ Index       |   233    |      1 Hz     |           [IAQ][3]                            |
+|                           |      Temperature      |   234    |      1 Hz     |  Measured at the position of the sensor       |
+|                           |        Humidity       |   235    |      1 Hz     |  Measured at the position of the sensor       |
+|                           |      AirPressure      |   236    |      1 Hz     |  Measured at the position of the sensor       |
+| **TK-SoundPressureLevel** |          SPL          |   237    |    100 Hz     |   Output db*10  [Sound Pressure Bricklet][4]  |
+|      **TK-Humidity**      |        Humidity       |   238    |               |   Output Rh% * 100 [Humidity v2][5]           |
+|      **TK-AnalogIn**      |        AnalogIn       |   239    |      100Hz    |                                               |
+| **TK-Particulate Matter** |          PM1.0        |   240    |      100Hz    |          Timestamped(int) [µg/m³][6]          |
+|                           |          PM2.5        |   241    |      100Hz    |          Timestamped(int) [µg/m³][6]          |
+|                           |          PM10         |   242    |      100Hz    |          Timestamped(int) [µg/m³][6]          |
+|     **TK-Dual0-20mA**     |      Solar-Light      |   243    |      100Hz    |          Timestamped(int) [mA x 1000000][7]   |
+|     **TK-Thermoouple**    |      Radiant Temp     |   244    |      100Hz    |          Timestamped(int) [°C x 100][8]       |
+|        **TK-PTC**         |        Air Temp       |   245    |      100Hz    |          Timestamped(int) [°C x 100][9]       |
+|        **ATMOS22**        |       North Wind      |   246    |      ~2Hz     |          Timestamped(float) [m/s][10]         |
```

Comment:
I feel we should probably create a `Units` column to report all of these and above sensor outputs.

Reply:
We will resolve this in a separate PR